bin/conjur-cli.rb

```diff
@@ -180,17 +198,32 @@ def test_select
 when this command is called, since it's used to encrypt the token-signing key
 in the database.
 
+The optional 'password_file' command argument is either a file which contains
+the custom password, or the dash '-' character, which signifies
+that the password should be read from stdin.
+
 Example:
 
-$ conjurctl account create myorg
+$ conjurctl account create myorg [password_file]
```

Comment:
Given it's optional, would it make since for this to be a flag (`--password-file`) instead? That also has the benefit of being consistent with the `server` command.

Reply:
this is small, but there is a difference between providing a reference to a file and to providing the file contents. should we be overloading the same flag with both use cases? maybe it would be better to have a `password` flag for the password value, and a `password-file` flag for a reference to a file containing the value?

Reply:
I am leaning towards just including a `password` flag (which would only accept input via STDIN). If there is value in adding the flag `password-file` I will be happy to add it in the future.